flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/table/ParameterizedPredicate.java

```diff
@@ -23,18 +23,35 @@
 import org.apache.commons.lang3.ArrayUtils;
 
 import java.io.Serializable;
+import java.util.ArrayList;
 
 /** A data class that model parameterized sql predicate. */
 @Experimental
 public class ParameterizedPredicate {
     private String predicate;
     private Serializable[] parameters;
+    private ArrayList<Integer> indexesOfPredicatePlaceHolders = new ArrayList<>();
+
+    public ArrayList<Integer> getIndexesOfPredicatePlaceHolders() {
```

Comment:
```suggestion
    public List<Integer> getIndexesOfPredicatePlaceHolders() {
```
It is recommened to use interfaces rather than concrete implementation